baiji/test_s3.py

```diff
@@ -26,9 +26,8 @@ def setUp(self):
         self.s3_path = loc.path
 
         b = s3.S3Connection()._bucket(self.bucket) # pylint: disable=protected-access
-        for key in b.list(self.s3_path[1:]):
-            b.delete_key(key)
-        self.assertEqual(len([x for x in b.list(self.s3_path[1:])]), 0) # just make sure we're starting with a clean slate
+        b.delete_key(self.s3_path[1:] + '.tempdir')
```

Comment:
What's the logic behind this? Do we know that this is the only file that might be there?